### PR TITLE
[RFC] Update triage process with more labels

### PR DESCRIPTION
**Pull request by thaJeztah**
Describe new labels in the triage process to improve the triage workflow / efficiency

/cc @icecrime

### CODE REVIEW

**Comment by icecrime** [comment]
I'm going to play devil's advocate here, because I really hoped we could have _less_ labels rather than _more_ (if only because it makes the whole thing more confusing for maintainers). These are the ones that I think could be removed without any loss of information:

- `status/possible-duplicate`: isn't _everything_ a possible duplicate until proven otherwise?
- `status/unconfirmed`: what's the difference between having `status/unconfirmed` and not having the label at all? It seems to me like the important information is carried by `status/confirmed`.
- `status/undecided`: what's the difference between having `status/undecided` and not having the label at all? It seems to me like the important information is carried by `status/accepted`.
- `status/help-needed`: I would argue that any issue is up to grab by the community by default, and that the `exp/*` labels help indicate a level of difficulty.

**Comment by thaJeztah** [comment]
@icecrime no problem at all, that's exactly the kind of feedback I was looking for. My train of thought with the "unconfirmed", "undecided" was that filtering on labels is easier than looking for comments; adding the label could be used to indicate "I looked at it, but needs more input to make the decision". I'm perfectly fine with leaving them out

w.r.t. The "help needed"; I know it came up a while back, so decided to add it here for discussion.
let's see what others think.

**Comment by icecrime** [comment]
@thaJeztah You can still filters with `-label:"status/confimed"`, and this will give you everything which isn't, no?

My problem with help-needed is the same than for every other label: people will forget to add it, or not think about it, and then what does it mean for the others? That community help isn't welcome? I'm going to caricature, but we're better off assuming that help is wanted on every issue, and have a `status/help-not-needed` label :wink:

**Comment by thaJeztah** [comment]
Updated, removed the "undecided", "possible-duplicate", "unconfirmed", and "help needed" labels

**Comment by tonistiigi** [comment]
One of the problems with current labels is that there is no way to mark an unconfirmed bug. For example, issues where someone run a documented command, got an unexpected result and there is nothing indicating why it happened. Usually, these end up being invalid on incomplete reports but there is still effort needed from both sides to finalize in this conclusion. It is wrong to mark those issues as bug(although we have done it in some cases) as it assumes something can be fixed while it usually can't. It also assumes bug was accepted and no other action is required from the reporter. Leaving the issue untriaged just wastes time as other maintainers don't know someone already looked at it and find out it can't be classified. We can just do `status/unconfirmed` with no kind, but I've heard that everything should have a `kind/*` label. So I'd like `kind/bug-maybe` or `kind/bug-unconfirmed`. Even `kind/unknown` would be improvement.

Reading the discussion, I don't agree about `status/help-needed`. This is not exp level or forbidding community to change certain areas. It's mainly stuff that has been proposed by community, we see no harm in them but don't see them important enough to invest our own time. Or these can be low priority confirmed bugs that have been stalled.

`status/accepted` seems to imply some kind of consensus between maintainers. Is it the same as 2 "design LGTMs"?

**Review by dnephin — COMMENTED** [review]
Minor suggestion for wording, otherwise LGTM

**Comment by dnephin** [comment]
`status/accepted` is for issues not PRs, so I think it's different.

**Review by icecrime — APPROVED** [review]
LGTM

File: project/ISSUE-TRIAGE.md [hunk 1]
```diff
@@ -105,6 +105,18 @@ written a whole plugin for Docker in a personal project and never contributed to
...
 class="gh-label expert">exp/expert</strong> level task.
 
+#### Triage status
+
+To communicate the triage status with other collaborators, you can apply status
+labels to issues. These labels prevent duplicating effort.
+
+| Status                        | Description                                                                                                                                                                 |
+|-------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| status/confirmed              | You triaged the issue, and was able to reproduce the issue. Always leave a comment how you reproduced, so that the person working on resolving the issue has a way to set up a test-case.
```
Comment (vdemeester) [inline project/ISSUE-TRIAGE.md]: @thaJeztah ping :angel:

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Pull request merged**